### PR TITLE
STR-47 -- FE: Login for admin users

### PR DESCRIPTION
**Pull request by BrianLinggadjaja**
# Description
Create admin route and logic; updated tests
  
# Testing
1. Log in as **`steve`**.
2. Ensure you are routed to admin panel.
3. Run **`yarn jest-unit`** and confirm all the test pass.

### CODE REVIEW

File: resources/js/components/dashboard/DashboardModules.vue [hunk 1]
```diff
@@ -10,7 +10,6 @@
 </template>
 
 <script>
-import { changeRouteTitle } from './../../mixins/changeRouteTitle.js'
 import DashboardNotification from './DashboardNotification'
 
 export default {
```
Comment (frlzjosh) [inline resources/js/components/dashboard/DashboardModules.vue:15]: make the module names consistent by capitalizing the first letter of each module name

File: resources/js/router.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Logout from './views/Logout.vue'
 import Dashboard from './views/Dashboard.vue'
 import Error from './views/Error.vue'
 import schedule from './views/schedule.vue'
+import admin from './views/admin.vue'
```
Comment (frlzjosh) [inline resources/js/router.js:9]: admin should capitalized, if you could fix schedule too that would be cool

File: resources/js/store/modules/user/actions.js [hunk 3]
```diff
@@ -7,21 +7,20 @@ export default {
...
           if (response.data.valid === '1') {
-            var cookieValue = response.data.token
-            var cookieExpirationDate = new Date()
+            let cookieValue = response.data.token
+            let cookieExpirationDate = new Date()
             cookieExpirationDate.setMonth(cookieExpirationDate.getMonth() + 1)
             document.cookie = `userKey = ${cookieValue}; expires = ${cookieExpirationDate.toUTCString()};`
             commit('VERIFY_USER_DATA', response.data)
-            router.push({ name: 'Dashboard' })
+
```
Comment (frlzjosh) [inline resources/js/store/modules/user/actions.js:15]: nice man!

**Review by frlzjosh — CHANGES_REQUESTED** [review]
Ok, so the functionality works. However I feel as if you should hide Modules to admin users by using a `v-if="!user.isAdmin"` the `Modules` listing on the navbar

**Review by frlzjosh — APPROVED** [review]
looks good!

File: tests/jsUnit/app.spec.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import 'jest-extended'
-import { shallowMount } from '@vue/test-utils';
-import App from './../../resources/js/App.vue';
+// import { shallowMount } from '@vue/test-utils';
```
Comment (luisjg) [inline tests/jsUnit/app.spec.js:2]: Are we not going to use this or is this for future use?

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**